lib/Controller/ApiController.php

```diff
@@ -186,21 +186,44 @@ public function getRooms() {
 	}
 
 	/**
-	 * @NoAdminRequired
+	 * @PublicPage
 	 * @NoCSRFRequired
 	 *
 	 * @param int $roomId
 	 * @return JSONResponse
 	 */
 	public function getPeersInRoom($roomId) {
 		try {
-			$room = $this->manager->getRoomForParticipant($roomId, $this->userId);
+			$room = $this->manager->getRoomById($roomId);
 		} catch (RoomNotFoundException $e) {
 			return new JSONResponse([], Http::STATUS_NOT_FOUND);
 		}
 
+		$participants = $room->getParticipants(time() - 10);
```

Comment:
Use `- 30`, -10 is too few as the signalling requests can take some time. I've added some more logic to the JS code tonight that makes -30 a fair choice.